### PR TITLE
db/rwd-buttons

### PR DESCRIPTION
**Pull request by karolkolodziej**
Added default styling to the rwd buttons, and added select state.

- [x] styling 
- [x] select
- [x] context menu style
- [x] unselect
- [x] test

### CODE REVIEW

File: css/dashboards/gui.css [hunk 1]
```diff
@@ -416,21 +416,17 @@
...
-.highcharts-dashboards-edit-context-menu-btn div {
-    height: 4px;
-    background-color: var(--highcharts-dashboards-neutral-color-50);
-    margin: 0 0 4px;
+    cursor: pointer;
 }
 
 .highcharts-dashboards-edit-context-menu-btn:hover {
-    border: 1px solid var(--highcharts-dashboards-neutral-color-80);
+    background-color: var(--highcharts-dashboards-neutral-color-50) !important;
```
Comment (pawellysy) [inline css/dashboards/gui.css:428]: Please use more accurate selector and avoid using `!important`

**Review by pawellysy — CHANGES_REQUESTED** [review]
One minor comment, otherwise good!

**Review by bre1470 — APPROVED** [review]
Looks good. 👍

File: css/dashboards/gui.css (hunk 1 above)
Comment (sebastianbochan) [reply css/dashboards/gui.css:428]: Yes, no `!important` anywhere, please!

File: ts/Dashboards/EditMode/EditMode.ts [hunk 2]
```diff
@@ -622,12 +622,31 @@ class EditMode {
...
-                            // reflow elements
+                        callback: (e: PointerEvent): void => {
+                            const button = e.target as HTMLElement,
+                                isSelected =
+                                    button.classList.contains('selected');
+
+                            // Deselect given button and reset board width.
+                            if (isSelected) {
+                                button.classList.remove('selected');
+                                this.board.layoutsWrapper.style.width = '100%';
```
Comment (sebastianbochan) [inline ts/Dashboards/EditMode/EditMode.ts:633]: You can remove the width from the style instead of applying 100%. It is by default in our CSS.

**Review by sebastianbochan — APPROVED** [review]

**Review by pawellysy — CHANGES_REQUESTED** [review]
image
On Darkmode, the context menu looks wrong 😢 

image


So does the rwd buttons

**Review by pawellysy — APPROVED** [review]
LGTM!

**Pull request merged**